pkg/keepkey/messages.go

```diff
@@ -237,25 +238,26 @@ func (kk *Keepkey) recoverDeviceRaw(numWords uint32, enforceWordlist bool) error
 // The useCharacterCipher flag tells the device to recover using the on-screen cypher or through entering
 // the words in a random order. This method must be called on an uninitialized device
 // Raw Mode provides a superior user experience but may not be available in all shell environments
-func (kk *Keepkey) RecoverDevice(numWords uint32, enforceWordList, useCharacterCipher, rawMode bool) error {
+func (kk *Keepkey) RecoverDevice(numWords uint32, enforceWordList, dryRun bool, useCharacterCipher, rawMode bool) error {
```

Comment:
drop the bool here. a lot of the time when you have a bunch of params with the same type in a row you only put the type once. Purely cosmetic doesn't change anything functionally

Reply:
drop the first one, or the last one, or both of them?

Reply:
the first one

Reply:
✅